src/network/networkd-netdev-gperf.gperf

```diff
@@ -102,3 +103,4 @@ Bridge.ForwardDelaySec,      config_parse_sec,                   0,
 Bridge.MulticastQuerier,     config_parse_tristate,              0,                             offsetof(Bridge, mcast_querier)
 Bridge.MulticastSnooping,    config_parse_tristate,              0,                             offsetof(Bridge, mcast_snooping)
 Bridge.VLANFiltering,        config_parse_tristate,              0,                             offsetof(Bridge, vlan_filtering)
+VRF.TableId,                 config_parse_unsigned,              0,                             offsetof(Vrf, table_id)
```

Comment:
unsigned? You defined the type as uint32_t, hence you could use config_parse_uint32 here, too..